noir-projects/aztec-nr/uint-note/src/uint_note.nr

```diff
@@ -233,12 +235,37 @@ impl PartialUintNote {
         context.push_note_hash(self.compute_complete_note_hash(value));
     }
 
-    fn compute_note_completion_log(self, value: u128) -> [Field; 2] {
+    /// Completes the partial note, creating a new note that can be used like any other UintNote.
+    pub fn complete_from_private(self, value: u128, context: &mut PrivateContext) {
```

Comment:
I don't perform to zero value check here like in public since it looks like we have the info of real private log length and hence the 0 value should not get trimmed. Can write a test for this if the reviewer is not sure if it's correct.